src/__internal__/popover/popover.component.tsx

```diff
@@ -35,6 +35,8 @@ export interface PopoverProps {
   animationFrame?: boolean;
   // Optional strategy to use for positioning the floating element. Defaults to "absolute".
   popoverStrategy?: "absolute" | "fixed";
+  // Allows child ref to be set via a prop instead of dynamically finding it via children iteration
+  childRefOverride?: MutableRefObject<HTMLDivElement | null>;
```

Comment:
**question**: would this functionality still work if `Popover` was a forward ref?

Reply:
I'm not really sure what you mean here, Popover doesn't actually render an element it either returns the content/ children or wraps them in a portal and returns that.